services/web-app/pages/about-carbon/case-studies/consistency-in-the-cloud.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright IBM Corp. 2021, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { NextSeo } from 'next-seo'
+
+import RemoteMdxLoader from '@/components/remote-mdx-loader'
+import { getRemoteMdxData } from '@/lib/github'
+
+const RemoteMdxPage = ({ source }) => {
+  const seo = {
+    title: 'Consistency in the Cloud'
+  }
+
+  return (
+    <>
+      <NextSeo {...seo} />
+      <RemoteMdxLoader source={source} />
+    </>
+  )
+}
+
+export const getStaticProps = async () => {
+  const mdxSource = await getRemoteMdxData(
+    {
+      host: 'github.com',
+      org: 'carbon-design-system',
+      repo: 'carbon-website',
+      library: 'carbon-website',
+      ref: 'main'
+    },
+    '/src/pages/case-studies/consistency-in-the-cloud.mdx'
```

Comment:
this one icon doesnt load for me, are we good with that? 
<img width="474" alt="image" src="https://user-images.githubusercontent.com/40550942/172718493-79b6b298-ffde-4176-bbf8-1ac8e608f1bf.png">

Reply:
Yea, remote svgs aren't loading, being tracked here https://github.com/carbon-design-system/carbon-platform/issues/767